.github/workflows/build-and-test.yml

```diff
@@ -637,12 +637,12 @@ jobs:
     needs: [ build-nodejs-Linux, train-test-model-Linux ]
     if: ${{ github.event_name == 'pull_request' }}
     strategy:
+      fail-fast: false
```

Comment:
why moving it up? it's adding noise to the patch

Reply:
(same for all the `fail-fast` changes)

Reply:
I think it has to do with git’s way to handle changes. Maybe by adding the new lines below, it’d make it cleaner…

Reply:
This was fixed, sorry for the noise :)